app/src/androidTest/java/org/simple/clinic/drugs/PrescriptionRepositoryAndroidTest.kt

```diff
@@ -11,6 +11,8 @@ import org.simple.clinic.TestData
 import org.simple.clinic.facility.Facility
```

Comment:
Can we reword the commit message?

Ex: "Add method for updating the prescribed drug frequency"